packages/account/src/Components/sent-email-modal/__tests__/sent-email-modal.spec.js

```diff
@@ -0,0 +1,158 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { isMobile } from '@deriv/shared';
+import SentEmailModal from '../sent-email-modal';
+
+const onClose = jest.fn();
+const onClickSendEmail = jest.fn();
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(),
+    isDesktop: jest.fn(),
+}));
+
+describe('<SentEmailModal/>', () => {
+    beforeAll(() => {
+        const modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should render SentEmailModal component to change dmt5 password', () => {
+        render(<SentEmailModal identifier_title='mt5' is_open onClose={onClose} onClickSendEmail={onClickSendEmail} />);
+
+        expect(
+            screen.getByText(/Please click on the link in the email to change your DMT5 password./i)
+        ).toBeInTheDocument();
+    });
+
+    it('should render SentEmailModal component to change deriv x password', () => {
+        render(
+            <SentEmailModal identifier_title='dxtrade' is_open onClose={onClose} onClickSendEmail={onClickSendEmail} />
+        );
+
+        expect(screen.getByText(/Deriv X/i)).toBeInTheDocument();
+    });
+
+    it('should render SentEmailModal component to change password through google account', () => {
+        render(
+            <SentEmailModal identifier_title='Google' is_open onClose={onClose} onClickSendEmail={onClickSendEmail} />
+        );
+
+        expect(
+            screen.getByText(/Check your Google account email and click the link in the email to proceed./i)
+        ).toBeInTheDocument();
+    });
+
+    it('should render SentEmailModal component to change email', () => {
+        render(
+            <SentEmailModal
+                identifier_title='Change_Email'
+                is_open
+                onClose={onClose}
+                onClickSendEmail={onClickSendEmail}
+            />
+        );
+
+        expect(screen.getByText(/Check your email and click the link in the email to proceed./i)).toBeInTheDocument();
+    });
+
+    it('should display default message when no appropriate identifier_title is passed', () => {
+        render(<SentEmailModal identifier_title='' is_open onClose={onClose} onClickSendEmail={onClickSendEmail} />);
+
+        expect(screen.getByText(/Please click on the link in the email to reset your password/i)).toBeInTheDocument();
+    });
+
+    it('should trigger onClose function when modal close button is clicked', () => {
+        render(<SentEmailModal identifier_title='mt5' is_open onClose={onClose} onClickSendEmail={onClickSendEmail} />);
+        const btn = screen.getByTestId('dt-send-email-template-close-test-id');
+        fireEvent.click(btn);
```

Comment:
Check please naming for test id. Accordingly to our last agreement with decided to use snake_case for data-testid names